### PR TITLE
Adding multiple-parallel pipeline

### PR DESCRIPTION
**Pull request by pallavipannu**

### CODE REVIEW

File: multiple-parallel/multiple-parallel.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+@kfp.dsl.pipeline(
+    name='Parallel stages pipeline',
+    description='An example pipeline to launch number of stages in parallel'
+)
+def parallel_pipeline():
+    count = 22
+    for i in range(count):
+        op = kfp.dsl.ContainerOp(name='parallel',image="alpine",command=["sleep", "5s"])
+        op.add_toleration(V1Toleration( effect='NoSchedule', key='node.kubernetes.io/unschedulable', operator='Exists'))
```
Comment (deepio-oc) [inline multiple-parallel/multiple-parallel.py]: what does this line do ?

Comment (pallavipannu) [reply multiple-parallel/multiple-parallel.py]: removed the line, robot test passed.

File: limits-requests/config.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+code:
+  name: dkube-examples
+  url: https://github.com/oneconvergence/dkube-examples.git
+  branch: tensorflow
+  private: false
+  
+arguments:
+  code: dkube-examples
+  run_script: stress --vm-bytes 10M
```
Comment (deepio-oc) [inline limits-requests/config.yaml]: to test limit requests work, the example needs to allocate more memory than the limit. Lets add a key at the end , "status: failed". Robot automation should check this key and expect pipeline to fail and not fail the test case

Comment (pallavipannu) [reply limits-requests/config.yaml]: added the status field, verified pipeline failed and robot checked status and it passed.

Comment (pallavipannu) [reply limits-requests/config.yaml]: allocated more memory and cpu, dkube giving error insufficient cpu and memory

File: arvados-s3/arvados-s3.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import json
```
Comment (deepio-oc) [inline arvados-s3/arvados-s3.py:1]: do we need test this pipeline with S3 remote option  where pipeline would download the dataset in one of the step.

For non-remote dataset where DKube downloads it , this pipeline does nothing specific to arvados-s3. Do we really need this pipeline in that case ?

File: limits-requests/config.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  branch: tensorflow
+  private: false
+  
+arguments:
+  code: dkube-examples
+  run_script: stress --vm-bytes 1000G
+  limit_cpu:  1000
+  limit_memory: 1000Gi
+  request_cpu: 999
+  request_memory: 999Gi
```
Comment (deepio-oc) [inline limits-requests/config.yaml]: these are too big limits and requests. Try with a reasonable limit and requests and then run_script should try to use more than limit and fail.

File: arvados-s3/arvados-s3.py (hunk 3 above)
Comment (deepio-oc) [reply arvados-s3/arvados-s3.py:1]: @mak-454

File: multimodel/multimodel-serving.py [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+            command="bash",
+            arguments=["-c", "ls /model"],
+            pvolumes={"/model": kfp.dsl.PipelineVolume(pvc="{{workflow.uid}}-model")},
+        ).after(storage)
+
+        serving = dkube_serving_op(
+            model=str(model),
+            device="cpu",
+            serving_image='{"image":"ocdr/inf-multimodel:latest"}',
+        ).after(storage)
```
Comment (deepio-oc) [inline multimodel/multimodel-serving.py:38]: @mak-454 does this pipeline reflects the multi-model usage by the customer?

File: limits-requests/config.yaml (hunk 4 above)
Comment (pallavipannu) [reply limits-requests/config.yaml]: Tried giving less values, 
Raised the issue https://github.com/oneconvergence/gpuaas/issues/7089

**Comment by deepio-oc** [comment]
@pallavi-pannu-oc can you close this?

**Pull request closed without merging**